StaticPagesSchemaMigration.php

```diff
@@ -38,15 +38,17 @@ public function up()
         Schema::create('static_page_settings', function (Blueprint $table) {
             $table->bigIncrements('static_page_setting_id');
             $table->bigInteger('static_page_id');
-            $table->foreign('static_page_id', 'static_page_settings_static_page_id')->references('static_page_id')->on('static_pages')->onDelete('cascade');
-            $table->index(['static_page_id'], 'static_page_settings_static_page_id');
+            $table->foreign('static_page_id', 'static_page_settings_static_page_id')
+                ->references('static_page_id')
+                ->on('static_pages')
+                ->onDelete('cascade');
+            $table->index(['static_page_id'], 'static_page_settings_static_page_id_index');
```

Comment:
Is this name change necessary? We use a similar convention to the previous index name elsewhere in the system -- see e.g. `lib/pkp/classes/migration/install/AnnouncementsMigration.php` -- without problems.

Reply:
fixed